### PR TITLE
feat(governance): add hardening recertification release

### PR DESCRIPTION
**Pull request by jamesryancooper**
Policy reference: `.octon/framework/agency/practices/pull-request-standards.md`
Reminder: Run `@codex review`.

## What

Implements the April 9, 2026 hardening recertification packet in canonical repo surfaces and promotes a new active release bundle, `2026-04-09-uec-hardening-recertification`.

## Why

The repository had the packet’s hardening changes implemented locally, but the release program was incomplete until a distinct hardening successor release existed and active governance/disclosure surfaces pointed at it. No-Issue: user-requested hardening recertification closeout.

## How

Added packet-specific validators and suites for lab reference integrity, host authority purity, runtime family depth, continuity linkage, release-known-limits, and retirement rationale. Normalized dossier/admission lab scenario references through an authored scenario registry and retained evidence index, promoted runtime artifact-depth disclosure into representative RunCards, generated the successor release bundle, and moved active lineage, closure, and effective projections onto the new release.

## Profile Selection Receipt

- Semantic version source(s): repo ingress and live governance surfaces
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `atomic`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): no support widening; active claim-bearing release changed; historical April 8 bundle retained; generated closure/disclosure surfaces rebased onto a new release id
- Tie-break status: no tie-break required
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): n/a

## Implementation Plan

- Workstreams and scope: lab integrity, authority purity, runtime artifact-depth validation, disclosure calibration, retirement rationale, active release promotion
- Public interfaces/types/contracts affected: claim-truth conditions, release-lineage, active HarnessCard, active closure status, validation workflows, support-target admissions, runtime/governance/behavioral/recovery suite registries
- Test scenarios: packet-specific local validators, release bundle regeneration, projection regeneration, recertification validation, default active-release resolution
- Assumptions/defaults: April 8 release remains historical after promotion; non-critical residual items stay explicit in residual ledger rather than blocking the release

## Impact Map (code, tests, docs, contracts)

- Code: assurance runtime scripts, governance scripts, workflows
- Tests: local validator sweep for packet-specific hardening checks and existing disclosure/parity checks
- Docs: prompt artifact, migration plan/evidence, active governance/disclosure wording
- Contracts: claim-truth conditions, constitutional family receipts, support-target admissions, retirement register, release lineage, active closure manifests

## Compliance Receipt

- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations

none

## Tradeoffs

Historical and exploratory references to the April 8 release remain in archived inputs and historical evidence. The PR promotes a new active release and keeps non-critical residuals open but explicit rather than trying to erase them.

## Testing

Local validation executed successfully:
- `bash .octon/framework/assurance/runtime/_ops/scripts/generate-release-bundle.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/runtime/_ops/scripts/generate-closure-projections.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/runtime/_ops/scripts/verify-support-dossier-parity.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/runtime/_ops/scripts/verify-lab-reference-integrity.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/runtime/_ops/scripts/verify-host-authority-purity.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/runtime/_ops/scripts/verify-runtime-family-depth.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/runtime/_ops/scripts/verify-continuity-linkage.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/runtime/_ops/scripts/verify-retirement-rationale.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/runtime/_ops/scripts/verify-release-known-limits.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/scripts/validate-known-limits-coherence.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/scripts/validate-claim-calibrated-disclosure.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-projection-byte-parity.sh 2026-04-09-uec-hardening-recertification`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-recertification-status.sh 2026-04-09-uec-hardening-recertification`

## Rollout

Promote `2026-04-09-uec-hardening-recertification` as the active release immediately on merge. The April 8 release remains retained historical evidence only.

## Convivial Purpose Check

- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric

- Risk class: [ ] Trivial [ ] Low [x] Medium [ ] High
- Rollback plan: restore active lineage and active closure/disclosure projections to the prior release bundle
- Rollback handle: `2026-04-08-uec-full-attainment-cutover`
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model

- OpenAPI/JSON-Schema changes: none
- Threat model update/link: claim-truth, authority-purity, and disclosure-calibration changes are captured in the updated governance and assurance contracts

## Observability and Performance

- Traces/logs/metrics for changed flows: release closure bundle, effective closure projections, runtime artifact-depth summaries, residual ledger
- Representative traces for high risk changes: new release closure reports under `/.octon/state/evidence/disclosure/releases/2026-04-09-uec-hardening-recertification/closure/`
- Performance or bundle impact: release bundle and validator footprint increased; no runtime-service performance change intended

## License and Provenance

- New dependencies and licenses: none
- Generated code/templates provenance notes: release bundle and active projections are regenerated from canonical repo surfaces

## Checklist

- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [ ] All review conversations resolved

## Screenshots / Notes

No UI changes. Historical release artifacts and exploratory packet references remain in-tree but are no longer the active claim-bearing surfaces.

### CODE REVIEW

File: .octon/framework/assurance/runtime/_ops/scripts/generate-closure-bundle.sh [hunk 1]
```diff
@@ -30,17 +37,22 @@ gate_failures=0
...
     "G6:capability-pack-consistency:validate-cross-artifact-capability-pack-consistency.sh" \
     "G7:route-consistency:validate-cross-artifact-route-consistency.sh" \
-    "G7A:run-contract-support-binding:../../../scripts/validate-run-contract-support-binding.sh" \
+    "G7A:run-contract-support-binding:verify-run-contract-support-binding.sh" \
     "G8:quorum-bindings:validate-quorum-policy-bindings.sh" \
     "G9:host-non-authority:validate-host-projection-non-authority.sh" \
     "G9A:canonical-authority-purity:../../../scripts/validate-canonical-authority-purity.sh" \
+    "G9B:host-authority-purity:verify-host-authority-purity.sh" \
     "G10:proof-plane-completeness:validate-proof-plane-completeness.sh" \
+    "G10A:lab-reference-integrity:verify-lab-reference-integrity.sh" \
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/assurance/runtime/_ops/scripts/generate-closure-bundle.sh:46]: Fixed in follow-up commits. `generate-closure-bundle.sh` now passes the target `release_id` into the release-aware `verify-*` gate scripts so non-active release generation evaluates and writes against the intended bundle instead of the active one.

File: .octon/framework/assurance/runtime/_ops/scripts/generate-closure-bundle.sh [hunk 2]
```diff
@@ -30,17 +37,22 @@ gate_failures=0
...
     "G11:wording-coherence:validate-disclosure-wording-coherence.sh" \
     "G11A:stage-attempt-disclosure-separation:../../../scripts/validate-stage-attempt-disclosure-separation.sh" \
-    "G11B:known-limits-coherence:../../../scripts/validate-known-limits-coherence.sh" \
+    "G11B:release-known-limits:verify-release-known-limits.sh" \
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/assurance/runtime/_ops/scripts/generate-closure-bundle.sh]: Fixed in follow-up commits. G11B is now wired back to `validate-known-limits-coherence.sh` as the blocking gate, while `verify-release-known-limits.sh` remains the report generator that materializes the residual-ledger-backed disclosure inputs.
